src/frontend/src/lib/types/token.ts

```diff
@@ -48,3 +48,5 @@ interface TokenFinancialData {
 export type TokenUi = Token & TokenFinancialData;
 
 export type TokenIndexKey = string;
+
+export type TokensTotalUsdBalancesPerNetwork = Record<NetworkId, number>;
```

Comment:
I would move this to a dedicated type module like `token-something.ts`.

`token.ts` currently define a single entity, maybe opiniated but just feels like it would be a bit cleaner what do you think?

Reply:
Agree! Moved to `token-balance.ts`.